Dockerfile

```diff
@@ -67,6 +67,11 @@ RUN \
 tar -xvzf /build/software/java/OpenJDK11U-jdk_x64_linux_hotspot_11.0.4_11.tar.gz --directory /build/software/java \
 && rm /build/software/java/OpenJDK11U-jdk_x64_linux_hotspot_11.0.4_11.tar.gz
 
+RUN \
```

Comment:
@Nirothipan @maheshika shall we align the `RUN` commands as per the suggestion (see https://github.com/wso2/build-docker/pull/13#discussion_r419457208) in the previous PR?